packages/nimble-components/src/select/styles.ts

```diff
@@ -28,6 +40,93 @@ export const styles = css`
     [part='end'] {
         display: contents;
     }
+
+    .listbox {
```

Comment:
Can you take a look at the Color theme styling? It doesn't look right -- the semi-transparent background color no longer goes all the way to the edges of the dropdown.

Published storybook:
![image](https://github.com/ni/nimble/assets/20542556/36b53a6b-ee38-4b07-b4c1-05cfbe721138)

This PR's storybook:
![image](https://github.com/ni/nimble/assets/20542556/e30c8346-0a27-43e1-846c-cdb3cec9ab78)

Reply:
Fixed.